saleor/warehouse/reservations.py

```diff
@@ -381,9 +382,10 @@ def is_reservation_enabled(settings) -> bool:
 
 
 def get_reservation_length(request) -> Optional[int]:
+    site = load_site(request)
```

Comment:
Similar to my previous comment, I think we should instead pass the site and whatever else is needed here as parameters instead of passing the entire request. Then we could keep `load_*` functions in the GraphQL layer.